onnxruntime/test/python/onnxruntime_test_float8_gemm8.py

```diff
@@ -89,7 +89,10 @@ def get_model_gemm(
         ]
         nodes = [n for n in nodes if n is not None]
         graph = make_graph(nodes, "gemm", inputs, [d], inits)
-        onnx_model = make_model(graph, opset_imports=[make_opsetid("", 19)], ir_version=9)
+        opset_imports = [make_opsetid("", 19)]
```

Comment:
Instead of hardcoding the version can we fetch the last released version? Looks like onnx does make it available: https://github.com/onnx/onnx/blob/b60f69412abb5393ab819b936b473f83867f6c87/onnx/defs/schema.h#L1091

Need to check the exposed api to fetch this though.

Reply:
I'll do that. I add `-1` to stay one release behind in case the user installs a recent version of onnx onnxruntime does not support yet.